mpi-proxy-split/mpi_plugin.cpp

```diff
@@ -636,8 +636,8 @@ getLhMmapList()
   if (fnc) {
```

Comment:
I suggest a small comment here, to clarify the key point:
`// Initialize g_numMmaps to point to lower half 'numRegions'`